docs/lake/design.md

```diff
@@ -424,7 +429,7 @@ and each entry contains a timestamp, branch references can be easily
 navigated by time.  For example, a list of branches of a pool's past
 can be created by scanning the branches log and stopping at the largest
 timestamp less than or equal to the desired timestamp.  Likewise, a branch
-can be located in a similar fashion then its corresponding commit object
+can be located in a similar fashion, then its corresponding commit object
```

Comment:
I'm not sure if the comma is strictly necessary, but before it was there I paused on this sentence too long thinking "then" should be "than".